woocommerce/woocommerce-gateway-stripe
# Update css class names with prefix 'wcstripe'

### Changes proposed in this Pull Request:
A small refactor to make the classnames consistent across the codebase. 

### Testing instructions
- Enable UPE related flags to view the new settings pages
- Check the onboarding wizard to make sure the UI is not broken
- Check the `WooCommerce -> Settings -> Payments -> Stripe Settings` to make sure the UI is not broken
